shoop/core/models/shops.py

```diff
@@ -40,3 +41,15 @@ class Shop(TranslatableModel):
 
     def __str__(self):
         return self.safe_translation_getter("name", default="Shop %d" % self.pk)
+
+    def create_price(self, value):
+        """
+        Create a price with given value and settings of this shop.
+
+        Takes the ``prices_include_tax`` setting into account, and in
+        future might also do the same for a currency setting.
```

Comment:
I assume `value` is just an unadorned decimal?

(`:type:` annotations would be nice.)